
class AuthenticationError(Exception):
    """Exception raised when an authentication failed."""


